proto/panacea/datadeal/v2/consent.proto

```diff
@@ -19,7 +19,7 @@ message Certificate {
 
 // UnsignedCertificate defines a certificate issued by an oracle as a result of data validation.
 message UnsignedCertificate {
-  string cid = 1;
+  string data_endpoint = 1;
```

Comment:
I'm confused as to whether this field is necessary. If we enforce a specification that all `consumer_service` store with the same key type(e.g. dataHash), `consumer` or `provider` can infer `data_endpoint` from the `deal`.

Reply:
IMO, we don't need `data_endpoint`. It can be obtained from `deal_id` and `data_hash` in consumer storage.

Reply:
I agree. We don't need `data_endpoint`.

Reply:
Thanks for your feedback. I remove it in [26f9f11](https://github.com/medibloc/panacea-core/pull/646/commits/26f9f1119f024c6fa65c73020a335ea2fd911bd6).